### PR TITLE
feat:ui for [entity]/[repo]/issues page

### LINKED ISSUE #156: UI /[entity]/[repo]/issues page
basic ui for the issues for the individual repo

### PR DESCRIPTION
**Pull request by e-roy**
### What does it do? Why?

👉🏻 &nbsp;Please add a description of what this `pull request` does.
This is a basic UI for the `[entity]/[repo]/issues` page closes #156 

### QA

👉🏻 &nbsp;Please add what you think the reviewer has to test.

### Review

- [ ] All GHA are success
- [ ] Use https://ui.shadcn.com/docs often as possible
      more to come...

### CODE REVIEW

File: src/app/[entity]/[repo]/issues/page.tsx
Comment (PeerRich) [inline src/app/[entity]/[repo]/issues/page.tsx:68]:
```suggestion
    <section className="py-6 m-auto">
```

Comment (PeerRich) [reply src/app/[entity]/[repo]/issues/page.tsx:68]: padding too wide. max-width not needed.

max-width should be applied on layout

**Comment by e-roy** [comment]
@PeerRich max-width on the layout might become restrictive with the github similar design.   we might want to consider removing padding on the layout.  

here is this page on mobile
repo-pulls

and here is this page [https://nostrgit.com/pulls](https://nostrgit.com/pulls) on mobile
mobile-pulls

**Comment by PeerRich** [comment]
@e-roy i can raise a new PR once this is merged

**Comment by e-roy** [comment]
> @e-roy i can raise a new PR once this is merged

that sounds great, thanks!

**Comment by cypherhoodlum** [comment]
Yeah @e-roy just ```git merge main``` to get the latest changes and resolve the conflicts if any, and then we can merge this when you feel the branch is ready :+1:

**Review by cypherhoodlum — APPROVED** [review]
This is a good starting point for the view :+1:

**Pull request merged**
